app/pages/ServiceDiscoveryResults/ServiceDiscoveryResults.tsx

```diff
@@ -167,9 +165,11 @@ const InnerServiceDiscoveryResults = ({
       >
         <Configure
           filters={`categories:'${algoliaCategoryName}'`}
-          aroundLatLng={userLatLng}
-          aroundRadius={searchRadius}
-          aroundPrecision={1600}
+          {...(category.disableGeoLocation !== true && {
+            aroundLatLng: userLatLng,
+            aroundRadius: searchRadius,
+            aroundPrecision: 1600,
+          })}
```

Comment:
I'm a little surprised, to be honest, that this is legal and doesn't cause a type error. `&&` is short-circuiting, meaning that if the left-hand side of it is `false`, the whole expression returns `false`, and I feel like `{...false}` should be illegal.

I guess if the linters and type checkers aren't yelling about this, then it's OK, but I feel like it might be more readable to pull the condition outside of the `<Configure>` component and to just duplicate the `filters` prop, e.g.:

```jsx
{category.disableGeolocation
  ? <Configure filters={`categories:'${algoliaCategoryName}'`} />
  : <Configure
      filters={`categories:'${algoliaCategoryName}'`}
      aroundLatLng={userLatLng}
      aroundRadius={searchRadius}
      aroundPrecision={1600}
    />
}
```

Reply:
Ahh, interesting. I've gone ahead and made your suggested change considering the confusion.